crates/bevy_render/src/texture/dds.rs

```diff
@@ -88,7 +88,7 @@ pub fn dds_format_to_texture_format(
                     TextureFormat::Bc3RgbaUnorm
                 }
             }
-            D3DFormat::A16B16G16R16 => TextureFormat::Rgba16Uint,
+            D3DFormat::A16B16G16R16 => TextureFormat::Rgba16Unorm,
```

Comment:
Checked the microsoft docs and it seems like A16B16G16R16 is an old dx9 format and the dx11 equivalent is DXGI_FORMAT_R16G16B16A16_UNORM, so this seems correct.